filters/filters.go

```diff
@@ -57,7 +68,7 @@ type FilterContext interface {
 	StateBag() map[string]interface{}
 
 	// Gives filters access to the backend url specified in the route or an empty
-	// value in case it's a shunt or loopback
+	// value in case it's a shunt, loopback. In case of dynamic backend initially empty.
```

Comment:
In case of dynamic backend initially empty. -> In case of dynamic backend initially empty and set by a filter.